package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
```

```suggestion
    "dev": "next dev -p ${PORT-3000}",
```

Missed the env port for the dev command